tests/wicked/before_test.pm

```diff
@@ -51,6 +48,24 @@ sub run {
         systemctl 'enable dhcpd.service';
         systemctl 'start dhcpd.service';
     }
+    if (check_var('WICKED', 'basic')) {
+        $self->get_from_data('wicked/check_interfaces.sh', '/data/check_interfaces.sh', executable => 1);
+        assert_script_run("rcwickedd restart");
+        if (check_var('IS_WICKED_REF', '0')) {
+            # Remove previous static config and leave dynamic one
+            assert_script_run("rm /etc/sysconfig/network/ifcfg-$iface");
+            $self->get_from_data("wicked/dynamic_address/ifcfg-$iface", "/etc/sysconfig/network/ifcfg-$iface");
```

Comment:
from url should be hardcoded because it is static file in data folder

Reply:
done